packages/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemModule.java

```diff
@@ -883,4 +891,37 @@ private void ensureDirExists(File dir) throws IOException {
       throw new IOException("Couldn't create directory '" + dir + "'");
     }
   }
+
+  /**
+   * Concatenated copy of org.apache.commons.io@1.4.0#FileUtils#forceDelete
+   * Newer version of commons-io uses File#toPath() under the hood that unsupported below Android SDK 26
+   * See docs for reference https://commons.apache.org/proper/commons-io/javadocs/api-1.4/index.html
+   */
+  private void forceDelete(File file) throws IOException {
+    if (file.isDirectory()) {
+      File[] files = file.listFiles();
+      if (files == null) {
+        throw new IOException("Failed to list contents of " + file);
+      }
+
+      IOException exception = null;
+      for (File f : files) {
+        try {
+          forceDelete(f);
+        } catch (IOException ioe) {
+          exception = ioe;
```

Comment:
Do we want to continue `forceDelete`ing files, or would we want to early throw here? Why one and not the other?

Reply:
The behaviour is copied from `commons-io` that we were using previously, so I don't want to change any logic introduced there.